x-pack/legacy/plugins/alerting/server/lib/get_create_task_runner_function.test.ts

```diff
@@ -85,23 +94,22 @@ test('successfully executes the task', async () => {
   const runnerResult = await runner.run();
   expect(runnerResult).toMatchInlineSnapshot(`
     Object {
-      "runAt": 2019-06-03T18:55:30.982Z,
+      "runAt": 1970-01-01T00:00:10.000Z,
```

Comment:
Interesting the date is set to such an old epoch-y date.  My only thought is that at some point in the future, if we ended up needing to calculate a _previous_ date from this, somehow, the epoch millis might go negative.  Which node should handle, but I wonder if some other part of the system might choke on negative numbers. 

```console
$ node -p 'new Date(-1)'
1969-12-31T23:59:59.999Z
```

OTOH, I guess would actually be a good test, so ... leave it in!?!?!?

Reply:
Yeah this seems to be what happens when you use fake timers, everything becomes `new Date(0)` by default 🤔